### ISSUE TITLE
i put too many door for the modem :(

### ISSUE THREAD

**Issue by enderman-2009** [cadergator10/opencomputer-security-system#27]
#  ***im sad :(***
![2023-01-19_20 10 58](https://user-images.githubusercontent.com/123119027/213596814-6546bb4d-675f-4c77-b925-3682a13cd200.png).

**Comment by cadergator10** [comment]
Yeah there ain't much I can do about this one at the moment. I could try cutting down on the size of the packets sent or possibly to split the doors into smaller packets in the future, but for the moment the best you can do is split the door into 2 different computers

**Comment by cadergator10** [comment]
There is a configuration option you can change in OpenComputers.cfg called "maxNetworkPacketSize"

```
# The maximum size of network packets to allow sending via network cards.   
# This has *nothing to do* with real network traffic, it's just a limit
# for the network cards, mostly to reduce the chance of computer with a
# lot of RAM killing those with less by sending huge packets. This does
# not apply to HTTP traffic.
maxNetworkPacketSize=8192
```


It doesn't hurt to increase it by a ton, but be aware that setting it way too high can cause issues with huge files that get transmitted

**Comment by enderman-2009** [comment]
you could try to send each door separtly but other than that idk

**Comment by enderman-2009** [comment]
but that would also probably require modifing the server too

**Comment by enderman-2009** [comment]
i also think it doesn't matter for regular computer cause it can't even have this many doors but it can matter for servers
